src/integrations/faq.md

```diff
@@ -112,6 +112,16 @@ Yes. Follow the instructions in the [Ledger support for Nyx documentation](https
 ### Where can I find network details such as deployed smart contract addresses? 
 In the [`network defaults`](https://github.com/nymtech/nym/blob/release/{{platform_release_version}}/common/network-defaults/src/mainnet.rs) file. 
 
+### What is Circulating Supply and how to find out the distribution amount?
+
+The Circulating Supply is the number of available `NYM` tokens in the Nyx blockchain system that are often used by exchanges.
+
+The up-to-date Validator API endpoints can be found via the [Swagger Documentation](https://validator.nymtech.net/api/swagger/index.html). There you will be able to find out using the following API calls to retrieve the correct distribution amount and circulating supply within Nym. 
+
+- GET `/circulating-supply`  
```

Comment:
these need context - what are these calls?? No need to put `GET` in there but people need to know that these are the functions that can be called to get this information